src/style.css

```diff
@@ -1,3 +1,86 @@
+:root {
+  --main-color: rgba(0, 172, 220, 1);
+  --header-height: 60px;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+ul,
+li,
+a {
+  all: unset;
```

Comment:
Using `all: unset;` can remove all default styling, which might affect accessibility and usability. Consider resetting only specific properties that need to be overridden, such as `list-style` for lists or `text-decoration` for links.